backend/src/services/group/group.entity.ts

```diff
@@ -76,6 +76,18 @@ class Group {
       permissionsIds,
     });
   }
+
+  public setName(name: string): void {
+    this.name = name ? name : this.name;
+  }
+
+  public setWorkersIds(workersIds: string[]): void {
+    this.workersIds = workersIds ? workersIds : this.workersIds;
+  }
+
+  public setPermissionsIds(permissionsIds: string[]): void {
+    this.permissionsIds = permissionsIds ? permissionsIds : this.permissionsIds;
+  }
```

Comment:
Why do we need this checks? I think it's better to remove them.
If you want to validate input data it's always better to explicitly throw an exception